go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.3.1
 	github.com/hashicorp/go-azure-helpers v0.65.0
-	github.com/hashicorp/go-azure-sdk v0.20240111.1094251
+	github.com/hashicorp/go-azure-sdk v0.20240111.1120634
```

Comment:
It's been mentioned frequently that the `go-azure-sdk` should always be bumped in a separate PR to avoid conflicts and the versions becoming outdated quickly, which has already happened in this case. Please remove this.